stream-chat-android-compose/detekt-baseline.xml

```diff
@@ -29,6 +28,7 @@
     <ID>MagicNumber:MediaAttachmentFactory.kt$0.25f</ID>
     <ID>MagicNumber:MediaAttachmentQuotedContent.kt$0.8f</ID>
     <ID>MagicNumber:MediaGalleryPreviewActivity.kt$MediaGalleryPreviewActivity$0.2f</ID>
+    <ID>MagicNumber:MediaGalleryPreviewActivity.kt$MediaGalleryPreviewActivity$0.5f</ID>
```

Comment:
?

Reply:
Detekt considers passing the default value parameter as a magic number.

We're using 0.5f as a fallback for fetching an extra float. This case should not occur, I'm not sure it's worth declaring a constant over it, however if you think it is I'll introduce one.

Reply:
🤦 get rekt